.github/workflows/integration-tests-v1.yaml

```diff
@@ -0,0 +1,87 @@
+name: Run integration tests for 1.0
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Environment to run the tests against'
+        type: choice
+        required: true
+        default: 'dev'
+        options:
+          - dev
+          - staging
+  workflow_call:
+    inputs:
+      environment:
+        default: 'staging'
+        required: false
+        type: string
+    secrets:
+      FIREBOLT_USERNAME:
+        required: true
+      FIREBOLT_PASSWORD:
+        required: true
+      FIREBOLT_STG_USERNAME:
+        required: true
+      FIREBOLT_STG_PASSWORD:
+        required: true
+      SERVICE_ID:
+        required: true
+      SERVICE_SECRET:
+        required: true
+      SERVICE_ID_STG:
+        required: true
+      SERVICE_SECRET_STG:
+        required: true
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ inputs.branch }}
```

Comment:
We can remove branch input altogether now since we don't have 0.x. It was in place to run workflows from master that checkout 0.x, since nightlies can't be scheduled from a branch. 
We'll still be able to choose branch from a dropdown when submitting the workflow and it will be picked up by the relevant steps implicitly.